layouts/shortcodes/note.html

```diff
@@ -0,0 +1,11 @@
+<!-- adapted from Docsy alert shortcode -->
+{{- $_hugo_config := `{ "version": 1 }` -}}
+{{ $color := "info" }}
+<div class="alert alert-{{- $color -}}" role="alert">
```

Comment:
@anishasthana @thesuperzapper Is there a way for us to make Note blue instead of grey ?
It looks like the grey note is very hard to see.
![Screenshot 2024-09-30 at 21 54 06](https://github.com/user-attachments/assets/1d0f89fa-b17c-4b87-b921-a860d771810b)

Reply:
![image](https://github.com/user-attachments/assets/17f30499-e995-486d-8e7f-7e8a347e7d1e)
Thoughts on this blue?

Reply:
Yeah, looks great!